File: CustomChance/Java/src/main/java/com/customchance/View/Styles.java
package com.customchance.View;

import java.awt.Color;
import java.awt.Font;

import com.spvessel.spacevil.Decorations.Style;
import com.spvessel.spacevil.Decorations.CornerRadius;
import com.spvessel.spacevil.Decorations.ItemState;
import com.spvessel.spacevil.Common.DefaultsService;
import com.spvessel.spacevil.Flags.SizePolicy;
import com.spvessel.spacevil.Flags.ItemAlignment;
import com.spvessel.spacevil.Flags.ItemStateType;

class Styles {
    private Styles() {
    }

    static Style getButtonStyle() {
        Style style = new Style();
        style.background = new Color(255, 181, 111);
        style.foreground = Color.black;
        style.borderRadius = new CornerRadius();
        style.font = DefaultsService.getDefaultFont(Font.BOLD, 14);
        style.width = 150;
        style.height = 30;
        style.widthPolicy = SizePolicy.FIXED;
        style.heightPolicy = SizePolicy.FIXED;
        style.setAlignment(ItemAlignment.HCENTER, ItemAlignment.TOP);
        style.setTextAlignment(ItemAlignment.HCENTER, ItemAlignment.VCENTER);
        style.addItemState(ItemStateType.HOVERED, new ItemState(new Color(255, 255, 255, 60)));
        return style;
    }

    static Style getLabelStyle() {
        Style style = Style.getLabelStyle();
        style.foreground = new Color(210, 210, 210);
        style.font = DefaultsService.getDefaultFont(Font.BOLD, 14);
        style.height = 30;
        style.heightPolicy = SizePolicy.FIXED;
        style.setAlignment(ItemAlignment.VCENTER, ItemAlignment.LEFT);
        style.setTextAlignment(ItemAlignment.LEFT, ItemAlignment.VCENTER);
        return style;
    }

    static Style getCommonContainerStyle() {
        Style style = new Style();

        style.background = new Color(54, 57, 63);
        style.widthPolicy = SizePolicy.EXPAND;
        style.heightPolicy = SizePolicy.EXPAND;
        style.setAlignment(ItemAlignment.LEFT, ItemAlignment.TOP);

        return style;
    }
}

File: Apps/UsefulApps/CustomChance/Net/View/Styles.cs
using System;
using System.Drawing;
using SpaceVIL;
using SpaceVIL.Common;
using SpaceVIL.Core;
using SpaceVIL.Decorations;

namespace CustomChance
{
    internal static class Styles
    {
        internal static Style GetButtonStyle()
        {
            Style style = new Style();
            style.Background = Color.FromArgb(255, 255, 181, 111);
            style.Foreground = Color.Black;
            style.BorderRadius = new CornerRadius();
            style.Font = new Font(DefaultsService.GetDefaultFont().FontFamily, 14, FontStyle.Bold);
            style.Width = 150;
            style.Height = 30;
            style.WidthPolicy = SizePolicy.Fixed;
            style.HeightPolicy = SizePolicy.Fixed;
            style.Alignment = ItemAlignment.HCenter | ItemAlignment.Top;
            style.TextAlignment = ItemAlignment.HCenter | ItemAlignment.VCenter;
            style.AddItemState(ItemStateType.Hovered, new ItemState(Color.FromArgb(30, 255, 255, 255)));
            return style;
        }

        internal static Style GetLabelStyle()
        {
            Style style = Style.GetLabelStyle();
            style.Foreground = Color.FromArgb(255, 210, 210, 210);
            style.Font = new Font(DefaultsService.GetDefaultFont().FontFamily, 14, FontStyle.Bold);
            style.Height = 30;
            style.HeightPolicy = SizePolicy.Fixed;
            style.Alignment = ItemAlignment.VCenter | ItemAlignment.Left;
            style.TextAlignment = ItemAlignment.Left | ItemAlignment.VCenter;
            return style;
        }

        internal static Style GetCommonContainerStyle()
        {
            Style style = new Style();

            style.Background = Color.FromArgb(255, 54, 57, 63);
            style.WidthPolicy = SizePolicy.Expand;
            style.HeightPolicy = SizePolicy.Expand;
            style.Alignment = ItemAlignment.Left | ItemAlignment.Top;

            return style;
        }
    }
}